sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -1350,12 +1394,10 @@ func jsonToSloBurnRateConditionBlock(conditions []TriggerCondition) map[string]i
 		switch condition.TriggerType {
 		case "Critical":
 			hasCritical = true
-			criticalAlrt["time_range"] = condition.TimeRange
-			criticalAlrt["burn_rate_threshold"] = condition.BurnRateThreshold
+			criticalAlrt = getAlertBlock(condition)
```

Comment:
doing this will not update the original `block[critical]` and `block[warning]`. Would recommend doing `block["critical"] = toSingletonArray(criticalAlrt)` after the for loop

Reply:
makes sense, updated